CesiumGltfContent/src/GltfUtilities.cpp

```diff
@@ -292,17 +294,45 @@ GltfUtilities::parseGltfCopyright(const CesiumGltf::Model& gltf) {
       continue;
     }
 
-    GltfUtilities::moveBufferContent(gltf, destinationBuffer, sourceBuffer);
+    assert(sourceBuffer.byteLength == int64_t(sourceBuffer.cesium.data.size()));
+    assert(
+        destinationBuffer.byteLength ==
+        int64_t(destinationBuffer.cesium.data.size()));
+
+    // Copy the data to the destination and keep track of where we put it.
+    // Align each bufferView to an 8-byte boundary.
+    size_t start = destinationBuffer.cesium.data.size();
+
+    size_t alignmentRemainder = start % 8;
+    if (alignmentRemainder != 0) {
+      start += 8 - alignmentRemainder;
+    }
+
+    destinationBuffer.cesium.data.resize(
+        start + sourceBuffer.cesium.data.size());
+    std::memcpy(
+        destinationBuffer.cesium.data.data() + start,
+        sourceBuffer.cesium.data.data(),
+        sourceBuffer.cesium.data.size());
+
+    sourceBuffer.byteLength = 0;
+    sourceBuffer.cesium.data.clear();
+    sourceBuffer.cesium.data.shrink_to_fit();
+
+    destinationBuffer.byteLength =
+        int64_t(destinationBuffer.cesium.data.size());
```

Comment:
This section is copied directly from `moveBufferContent`. I couldn't think of a clean way to continue to use `moveBufferContent` while still being performant. Not sure how much use `moveBufferContent` is anymore...